src/main/java/gov/ornl/ssm/ml/ui/views/TrainingView.java

```diff
@@ -85,82 +100,102 @@ public void init() {
 
 		// Json representation of the models
 		String jsonModels = "";
+		
+        // Get the current user
+        Authentication authentication = SecurityContextHolder.getContext()
+                .getAuthentication();
+		
+        // The user information
+        OidcUser user = (OidcUser) authentication.getPrincipal();
+        OAuth2AuthorizedClient client =
+                authorizedClientService.loadAuthorizedClient("keycloak", user.getName());
 
 		// List of all models from the CURIES database
 		models = new ArrayList<Model>();
 		try {
 			try {
 
 				// Get the model digest from the backend
-				URL url = new URL(config.getFusekiHost() + "/api/datasets/curies/models?pageSize=220");
+				URL url = new URL(config.getFusekiHost() + "/api/datasets/test/models?pageSize=220");
```

Comment:
So the catalog can hold multiple "collections" (i.e. `/datasets/<collection>`).

Can we switch to allowing the User to pick different collection names instead of hard-coding it? (i.e. select either `test` or `curies`)

Reply:
For now, I've made it configurable at deploy time. We can discuss what the proper way of handling multiple collections should be, since I would assume people would want to be able to train a single model using files from all relevant collections.

Reply:
Agree and you are correct.

Thank you for adding the configuration at deploy time, happy with that!